website/docs/r/google_kms_key_ring.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "google"
+page_title: "Google: google_kms_key_ring"
+sidebar_current: "docs-google-kms-key-ring-x"
```

Comment:
why x?

Reply:
The file I was using as a reference had it as well.  I'll remove it.